aws-integration/trace-sample-app/values.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: sample-app
```

Comment:
These labels are a bit uncommon. Follow this: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/